# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../rrdtool/rrdtool-1.2.19-parsetime.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

parsing absolute time should not send us 30,000 years into the future
(Tatsuki Makino tatsuki_makino hotmail.com)

--- ./src/parsetime.c	(Revision 1001)
+++ ./src/parsetime.c	(Revision 1002)
@@ -850,7 +850,7 @@
 	      if ( ptv->tm.tm_hour == 30 ){
 		ptv->tm.tm_hour = hour_sv;
               }
-	      if ( ptv->tm.tm_hour == 30000 ){
+	      if ( ptv->tm.tm_year == 30000 ){
 		ptv->tm.tm_year = year_sv;
               }
 	    };	    
